source/docs/software/commandbased/organizing-command-based.rst

```diff
@@ -269,7 +312,15 @@ Then, elsewhere in our code, we can instantiate an single instance of this class
 
   .. code-block:: c++
 
-    // TODO
+    AutoRoutines autoRoutines{drivetrain, intake};
```

Comment:
This is trying to pass references/value to a pointer parameter and won't compile; use `&`